### DISCUSSION TITLE
Source map files for web-extension

### DISCUSSION THREAD

**Discussion by avalanche1** [parcel-bundler/parcel#8523] [Q&A]
How can I generate Source map files for web-extension platform?
It always gets generated inline (no matter what options I set in `targets` in `package.json` - and I, of course, don't want them inline in my production build - instead I need to upload `.map` files to Sentry.

**Comment by avalanche1** [comment]
@mischnic I really need your help with this 🙏

**Reply by mischnic** [reply]
Looks like this is caused by these hardcoded overrides:
https://github.com/parcel-bundler/parcel/blob/de1569572ab681bb6e90d2b330d002b5df7947ea/packages/transformers/webextension/src/WebExtensionTransformer.js#L387-L408

cc @101arrowz

**Reply by 101arrowz** [reply]
Source maps don't actually work when they aren't inline in web extensions but we should allow this feature to be overridden when needed. Haven't been too active on maintaining webext in parcel lately but I think this should be an easy fix for anyone else.

**Comment by avalanche1** [comment]
https://github.com/parcel-bundler/parcel/issues/8530
